frontend/src/app/application-dashboard/models/metrics.model.ts

```diff
@@ -1,5 +1,6 @@
 export interface MetricsDto {
```

Comment:
pls change to a more meaningful name since PageDto has disappeared. I would suggest something like PageWithMetricsDto